indra/llwebrtc/llwebrtc.cpp

```diff
@@ -875,6 +874,13 @@ bool LLWebRTCPeerConnectionImpl::initializeConnection(const LLWebRTCPeerConnecti
                 codecparam.parameters["stereo"]       = "1";
                 codecparam.parameters["sprop-stereo"] = "1";
                 params.codecs.push_back(codecparam);
+
```

Comment:
Reduce CPU usage on the client and server at the cost of bandwidth.  It'll allow us to get more clients on the server.